### ISSUE TITLE
Dashicon not loading properly in test environment when loaded from externals.

### ISSUE THREAD

**Issue by BE-Webdesign** [WordPress/gutenberg#944]
The Dashicon is not loading properly in the test environment for a build.  I think somehow wp.element is not being added in as an external which Dashicon requires to extend the component class.

**Comment by BE-Webdesign** [comment]
@nylen 

:eyes:

**Comment by nylen** [comment]
I see it :|

Probably caused by #929 somehow.

What error are you getting here?

**Comment by BE-Webdesign** [comment]
Yeah it is caused by #929, but not to worry I think it can be resolved fairly easily.

Here is the warning message.
```
Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in.
```

Dashicon extends wp.element

We need to add the externals into the test environment, the only hangup I have is figuring out how to use node externals and another library in the same go.  Was curious if you know how to do that.

**Comment by nylen** [comment]
Noting that this will be fixed by the current state of #935 (https://github.com/WordPress/gutenberg/pull/935/files#diff-512a8a050d031aaf7a1c47fb4f2a9656R10).

**Comment by youknowriad** [comment]
closing as fixed
